src/components/ui/fixed-toolbar-buttons.tsx

```
@@ -0,0 +1,71 @@
+  SubscriptPlugin,
+  SuperscriptPlugin,
+} from '@platejs/basic-nodes/react';
+import {
+  Bold,
+  Italic,
+  Underline,
+  Strikethrough,
+  Code,
+  Subscript,
+  Superscript,
+} from 'lucide-react';
```

ESLint

[error] Replace `⏎··Bold,⏎··Italic,⏎··Underline,⏎··Strikethrough,⏎··Code,⏎··Subscript,⏎··Superscript,⏎` with `·Bold,·Italic,·Underline,·Strikethrough,·Code,·Subscript,·Superscript·`

(prettier/prettier)